helpers/getFirstParagraph.ts

```diff
@@ -1,4 +1,4 @@
-const getFirstParagraph = (content: any) => {
+const getFirstParagraph = (content: any): string | null => {
```

Comment:
This is a correct return type for this function?

Previously, you told me it is hard to get type for this function, so I'm just curious

Reply:
I told about argument because this function accepts parsed AST from tina, so I don't have type for it